lib/run-script-pkg.js

```diff
@@ -41,7 +47,15 @@ const runScriptPkg = async options => {
   )
     cmd = defaultGypInstallScript
   else if (event === 'start' && await isServerPackage(path))
-    cmd = 'node server.js' + args.map(a => ` ${JSON.stringify(a)}`).join('')
+    // `node` probably means `C:\Program Files\nodejs\node.exe`.
+    // But it can also point to `.\node_modules\.bin\node.cmd` if you install a
+    // Node.js version using `npm i node`.
+    // So it might point to an executable, or a batch file. They require
+    // different escaping. Also, `puka` only supports batch-style escaping.
+    // The solution is to always use a batch file on Windows.
+    cmd = isWindows
+      ? sh`${pathModule.join(__dirname, 'node.cmd')} server.js ${args}`
+      : sh`node server.js ${args}`
```

Comment:
Note: This means that `npm start` with no `"start"` in package.json but with a `server.js` will print something like this on Windows:

```
> C:\absolue\path\to\npm\node_modules\run-script\lib\node.cmd server.js
```

instead of:

```
> node server.js
```

Let me know if you’d rather want to _print_ `> node server.js` on Windows too (even though that’s not what we’re actually executing).